package.json

```diff
@@ -35,7 +35,7 @@
     "build": "sl-scripts build",
     "build.docs": "sl-scripts build:typedoc",
     "commit": "git-cz",
-    "lint": "tslint 'src/**/*.ts'",
+    "lint": "tslint -p tsconfig.json 'src/**/*.ts'",
```

Comment:
Why is this required?

Reply:
By giving the tsconfig file tslint has types informations which some rules require — such as `no-floating-promises`

Reply:
Ah, makes sense. I thought it would just default to it. Nice.

Reply:
I thought the same to be honest!